cmd/brew-test-bot.rb

```diff
@@ -446,6 +446,8 @@ def download
         diff_end_sha1 = diff_start_sha1 = current_sha1
       end
 
+      ohai "Considering testing commits #{diff_start_sha1} to #{diff_end_sha1}"
```

Comment:
This looks weird if it is empty: https://dev.azure.com/Homebrew/Homebrew/_build/results?buildId=9315&view=logs&jobId=a5e52b91-c83f-5429-4a68-c246fc63a4f7&taskId=00f564b5-c6ae-5329-43c9-96bc79a72ec9&lineStart=33&lineEnd=33&colStart=1&colEnd=44

Reply:
I don't think `ohai` makes sense for these as they are otherwise only used for commands and it looks messy/confusing to have these be the only non-command values.

Reply:
@MikeMcQuaid is there something like `Debug`? that we could turn on/off when needed? Or would a simple `puts` make the job?

Reply:
`puts` is fine but let's figure out a way to combine it with https://github.com/Homebrew/homebrew-test-bot/pull/258#discussion_r283655711